app/helpers/application_helper.rb

```diff
@@ -205,8 +205,8 @@ def searchable?
   end
 
   def auto_complete_search(name, val, options = {})
-    path = options[:path] || send("#{controller_name}_path")
-    options.merge!(:class => "autocomplete-input form-control", :'data-url' => "#{path}/auto_complete_#{name}" )
+    path = options[:path] || send("#{controller_name}_path") + "/auto_complete_#{name}"
```

Comment:
Yes, although it does change the meaning of :path a little, see my last comment before you changed this.